Scripts/Scripts.csproj

```diff
@@ -0,0 +1 @@
+<Project Sdk="Microsoft.Build.NoTargets/3.1.0" />
```

Comment:
Isn't it a bit overkill to have all these different projects?
It may not be the case anymore but visual studio's performance used to degrade quite drastically based on the amount of projects loaded.

Reply:
I work with another team who uses projects this way. It seems like a decent alternative to managing solution folders. I am pretty sure either VS2022, or 2019 claimed to fix the performance issues with large numbers of projects.